run_dir/design/flowcell.html

```diff
@@ -172,49 +182,96 @@
                         warning
                         {% end %}
                     ">{% for thousand in sample.get('readsnb', sample.get('clustersnb', '')).split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}</td>
-                    <td class="text-left {% if sample_q30 < 40.0 %} danger
+                    
+                    {% if flowcell.get('two_reads') %}
+                      {% set overthirty_r1 = float(sample.get('overthirty_r1')) %}
+
+                      <td class="text-center {% if sample.get('overthirty_r1') < 40.0 %} danger
+                          {% elif overthirty_r1 < q30_threshold %} warning
+                          {% else %} success {% end %}">{{ sample.get('overthirty_r1') }}</td>
+                        
+                      {% set overthirty_r2 = float(sample.get('overthirty_r2')) %}
+                      <td class="text-center {% if overthirty_r2 < 40.0 %} danger
+                          {% elif overthirty_r2 < q30_threshold %} warning
+                          {% else %} success {% end %}">{{ sample.get('overthirty_r2') }}</td>
+                    {% else %}
+                      <td class="text-center {% if sample_q30 < 40.0 %} danger
```

Comment:
Better to use hyphens in the `{% if something -%}` blocks to avoid line breaks in this declaration, or just drop the line breaks if you prefer:

```
{% if sample_q30 < 40.0 %}danger{% else %}success{% end %}
```

It's a bit risky to break HTML tags across multiple lines.